assets/airports/ksav.json

```diff
@@ -0,0 +1,1899 @@
+        "JROSS":  ["N32d42m40.008", "W80d37m37.990"],
+        "KUCES":  ["N32d42m40.008", "W80d37m37.990"],
+        "KSAV":   ["N32d07m39.298", "W81d12m07.675", "Savannah"],
+        "LOTTS":  ["N32d20m11.534", "W81d51m17.258"],
+        "LUCNE":  ["N32d49m46.236", "W81d12m24.162"],
+        "MAYAR":  ["N32d14m38.620", "W81d32m03.949"],
+        "MIZLU":  ["N32d01m04.134", "W81d11m57.267"],
+        "MQQ":    ["N31d49m52.344", "W81d30m36.648", "Mc Intosh"],
+        "NOMAS":  ["N32d20m54.130", "W80d52m35.932"],
+        "PLZZZ":  ["N32d43m50.003", "W81d13m05.983"],
+        "REBUS":  ["N32d27m44.115", "W81d00m12.329"],
+        "RICKY":  ["N32d04m15.777", "W81d05m38.129"],
+        "SAV":    ["N32d08m46.563", "W81d11m56.773", "Savannah"],
+        "SINBY":  ["N32d07m38.890", "W81d05m06.814"],
+        "SMALZ":  ["N32d28m33.003", "W81d31m40.991"],
+        "SOOOP":  ["N32d22m22.996", "W80d33m50.983"],
+        "SOVIE":  ["N32d07m47.138", "W81d20m09.427"],
+        "SPONG":  ["N32d29m36.132", "W81d23m13.036"],
+        "SUTME":  ["N32d20m03.799", "W81d12m05.944"],
+        "SVN":    ["N32d00m42.999", "W81d08m27.095", "Hunter"],
+        "TELOE":  ["N32d07m34.354", "W80d58m02.690"],
+        "TERBE":  ["N31d54m44.803", "W81d11m54.410"],
+        "TILEE":  ["N32d07m50.393", "W81d27m14.074"],
+        "TILLS":  ["N32d28m44.099", "W81d14m04.844"],
+        "TOSEA":  ["N32d14m03.037", "W81d12m03.199"],
+        "VIYAP":  ["N31d15m08.147", "W81d26m08.188"],
+        "VP334":  ["N32d02m50.000", "W81d43m58.700"],
+        "VPBFE":  ["N31d15m08.147", "W81d26m08.188"],
+        "VPIWA":  ["N31d48m19.998", "W81d25m50.989"],
+        "VPZIE":  ["N32d01m36.996", "W80d53m24.982"],
+        "WADEB":  ["N32d07m55.529", "W81d41m22.358"],
+        "WISPR":  ["N32d01m58.857", "W81d13m30.705"],
+        "WIUMS":  ["N32d10m46.767", "W80d46m05.091"],
+        "WOHPY":  ["N31d50m16.922", "W81d12m37.201"],
+        "WORIB":  ["N32d02m33.756", "W80d58m07.305"],
+        "WOSUG":  ["N31d54m42.722", "W81d17m46.852"],
+        "WOXIT":  ["N32d12m51.009", "W81d27m11.107"],
+        "YUHVY":  ["N32d11m41.727", "W81d07m18.269"],
+        "ZALAV":  ["N32d08m24.287", "W80d49m24.824"],
+        "ZUTAN":  ["N31d54m46.609", "W81d06m01.969"]
+    },
+    "runways":[
+        {
+            "name": ["10", "28"],
+            "end": [
+                ["N32d07m43.508", "W81d13m07.690", "10ft"],
+                ["N32d07m42.503", "W81d11m18.920", "10ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["19", "1"],
+            "end": [
+                ["N32d08m08.967", "W81d12m00.511", "10ft"],
+                ["N32d06m59.665", "W81d11m59.981", "10ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "spawnPatterns": [
+        {
+            "origin": "KSAV",
+            "destination": "",
+            "category": "departure",
+            "route": "KSAV..SMALZ..CESOR",
```

Comment:
You don't actually need to define KSAV as part of their route here. Alternatively you can add KSAV but make sure to include the departure rwy.

For example:
> "KSAV10..SMALZ..CESOR"

It would also be good to give departures a heading to fly after takeoff. This can be done with the commands key, which you read more about here: https://github.com/openscope/openscope/blob/develop/documentation/spawnPatternReadme.md

Reply:
Fixed